apps/web/src/api/organization.ts

```diff
@@ -12,7 +12,7 @@ export function getCurrentOrganization() {
 export function inviteMember(email: string) {
   return api.post(`/v1/invites`, {
     email,
-    role: MemberRoleEnum.ADMIN,
+    role: MemberRoleEnum.MEMBER,
```

Comment:
The only thing i would change is to still invite an Admin by default, for now we don't have a well established permissions and roles management in terms of what a member can or cannot do. It dissalows to delete certain templates and invite new members. During this staged I would prefer to go with a more permissive approach of ADMIN by default as for now it's more relevant for most of the cases.

When we will do a more refined Access control we can change the default behaviour, or prompt the user on what he want to invite as.

Reply:
Yeah, understandable 🙂